### PR TITLE
Stored the result of get_chat_message_history() in a variable for reuse in clear_chat_history() function

### PR DESCRIPTION
**Pull request by Abhinavcode13**

### CODE REVIEW

File: gpt_computer_assistant/agent/chat_history.py [hunk 1]
```diff
@@ -20,5 +21,6 @@ def get_chat_message_history():
 
 
 def clear_chat_history():
+    chat_history = get_chat_message_history()
```
Comment (Mideky-hub) [inline gpt_computer_assistant/agent/chat_history.py:24]: chat_history here is storing connection, not history_db_path.

File: gpt_computer_assistant/agent/chat_history.py [hunk 2]
```diff
@@ -9,7 +9,8 @@
 
 
 def get_chat_message_history():
-    print("HISTORY DB", get_history_db())
+    history_db_path = get_history_db()
```
Comment (Mideky-hub) [inline gpt_computer_assistant/agent/chat_history.py:12]: create an object which contain a logrotate logging system for chat history in a compressed pipeline, then retreive those with sha256 if you really want to do it clean. but overall, i don't see the relevant purpose of the proposal

**Review by Mideky-hub — COMMENTED** [review]
Can you explain further the relevancy of creating a variable not assiacted with a class nor an object, not returning it, assign a connection to a chat_history variable to not use it? I may be missing something 🤔

**Comment by onuratakan** [comment]
Thank you but not a improvement.

**Comment by Abhinavcode13** [comment]
> Thank you but not a improvement.

Cool, any you look at approved PR #50.

**Pull request closed without merging**